apps/laboratory/src/utils/EIP5792Utils.ts

```diff
@@ -19,7 +26,11 @@ export function getAtomicBatchSupportedChainInfo(
     const walletCapabilities = walletCapabilitiesString && parseJSON(walletCapabilitiesString)
     const accountCapabilities = walletCapabilities[address]
     const chainIds = accountCapabilities
-      ? Object.keys(accountCapabilities).map(chainIdAsHex => Number(chainIdAsHex))
+      ? Object.keys(accountCapabilities)
+          .filter(
+            chainIdAsHex => accountCapabilities[chainIdAsHex]?.[capability]?.supported === true
```

Comment:
do we need to check for `true` explicitly? 
is the expected type other than `boolean?`

Reply:
expected type is boolean, but because it getting serialized and some might set it as a string "true", but from ERC it specifically say to ignore if it not boolean.
>The only valid JSON-RPC values for this capability’s only member, supported, are true or false; if a returned value for supported is typed as a string or number, it SHOULD be considered malformed.